src/Event/Events.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Event;
+
+final class Events
+{
+    const PRE_READ = 'api_platform.pre_read';
+    const POST_READ = 'api_platform.post_read';
+
+    const PRE_WRITE = 'api_platform.pre_write';
```

Comment:
Can we put them in the correct order? Read -> Deserialize -> Validate -> Write -> Serialize